### DISCUSSION TITLE
Anyone else have to restart the app completly after file changes?

### DISCUSSION THREAD

**Discussion by ComatoseHub** [vladmandic/sdnext#3467] [Q&A]
Every time I generate images or download a model or something from civit/hugging I have to shutdown the server completely and run it again to get it to be available in the UI. Curious if anyone else has seen this and/or found a solution.

I am on Windows with ZLUDA.

**Comment by vladmandic** [comment]
refresh button?

**Reply by ComatoseHub** [reply]
Refreshing the page does nothing and restarting the UI from system also doesn't load anything.

Sorry forgot to mention this as well.

**Reply by vladmandic** [reply]
i asked about refresh button which exists next to model selection and in extra networks. not reloading or restarting the ui.

**Reply by ComatoseHub** [reply]
Yep apologies, poor wording on my part I have tried that button as well. The only way I am able to see the file changes is if I restart the server completely. If I generate images nothing shows in the gallery until restart and loras and models don't show up until after a restart as well.

**Reply by vladmandic** [reply]
cannot reproduce, please create an issue and document fully.
focus on either model, lora or gallery images, not all 3 at the same time as those are very different.

**Reply by vladmandic** [reply]
to be exact model/lora is common, gallery is very different. you've created issue for gallery, that is ok.
lets have a separate issue for model/lora.
